job-dsls/jobs/communityRelease_pipeline.groovy

```diff
@@ -489,10 +489,17 @@ def kieWbTest='''#!/bin/bash -e
 wget -q https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/kie-wb-distributions/$kieVersion/kie-wb-distributions-$kieVersion-project-sources.tar.gz  -O sources.tar.gz
 tar xzf sources.tar.gz
 mv kie-wb-distributions-$kieVersion/* .
-rmdir kie-wb-distributions-$kieVersion'''
+rmdir kie-wb-distributions-$kieVersion
+
+echo "KIE version $kieVersion - kie-wb-common"
+wget -q https://repository.jboss.org/nexus/content/repositories/kieAllBuild-$baseBranch/org/kie/workbench/kie-wb-common/$kieVersion/kie-wb-common-$kieVersion-project-sources.tar.gz -O sources.tar.gz
```

Comment:
the URL should be like:
https://repository.jboss.org/nexus/content/groups/kie-groups/....

Reply:
Yes, I follow that pattern now, thank you